app/src/main/java/com/spendster/presentation/signUp/SignUpPresenter.java

```diff
@@ -28,8 +37,28 @@ public void signUp(String email, String password, String retypePassword){
             }
         }
         else {
-            if (signUpView != null) {
-                signUpView.showNextActivity();
+            if (signUpModel != null){
+                signUpModel.getUser(email, password)
```

Comment:
@SimonMaha should be used with disposable http://reactivex.io/RxJava/javadoc/io/reactivex/disposables/CompositeDisposable.html